modules/stochastic_tools/doc/content/syntax/ParameterStudy/index.md

```diff
@@ -0,0 +1,160 @@
+
+1. Defining parameters and quantities of interest (QoIs) within the given physics input.
+1. Perturbing the parameters using the specified sampling method.
+1. Efficiently running each instance of the perturbation.
+1. Gathering the QoI results.
+1. Computing statistics on the QoIs
+1. Outputting the QoIs and statistics.
+
+## Defining the Study
+
+The problem containing the physics is defined via another input file and specified using the [!param](/ParameterStudy/input) parameter. The idea is that this input defines the "nominal" case of the problem; as such, it should be able to run independently (without the `ParameterStudy` block).
+
+The parameters being perturbed in the study are specified using the [!param](/ParameterStudy/parameters) parameter. This is a list of parameters in the physics input that use [CommandLine.md] syntax.
+
+The QoIs for the study are specified using the [!param](/ParameterStudy/quantities_of_interest) parameter. This is a list of [reporter](Reporters/index.md) values with `<object_name>/<value_name>` syntax. [Postprocessor](Postprocessors/index.md) values use `<pp_name>/value` syntax. [Vector-postprocessor](VectorPostprocessors/index.md) values use `<vpp_name>/<vector_name>` syntax. This parameter is optional as users may want to do their own analysis using outputs, such as CSV or Exodus, generated from the physics input.
+
+Whether or not to computate statistics on the QoIs is defined by the [!param](/ParameterStudy/compute_statistics) parameter. The default for this parameter is `true`, so statistics will be computed by default if QoIs are specified with the [!param](/ParameterStudy/quantities_of_interest) parameter. A [StatisticsReporter.md] object is created, which computes the mean and standard deviation of the QoIs with 90%, 95%, and 99% confidence intervals.
+
+## Sampling Method id=sec:sampling
+
+There are various methods of sampling included with the `ParameterStudy` syntax, including random and pre-defined techniques described below. This method is specified using the [!param](/ParameterStudy/sampling_type) parameter.
+
+### Random Sampling id=sec:random
+
+The `monte-carlo` and `lhs` options in [!param](/ParameterStudy/sampling_type) implement random sampling using the [MonteCarloSampler.md] and [LatinHypercubeSampler.md] samplers, respectively. [!param](/ParameterStudy/num_rows) is a required parameter that defines the total number of samples. [!param](/ParameterStudy/distributions) is also a required parameter that defines the probablity distribution for each parameter. The available distributions are described in the following subsection. Each distribution type have their own set of required parameters that are specified via a list of values. For example, defining a random sampling with three parameters using `uniform`, `normal`, and `uniform` distributions is shown below:
```

Comment:
The "rows" nomenclature was being used before I joined. I think that was chosen since samplers were visualized as a matrix where each row is like a sample. I think I would like to keep the "rows" nomenclature for two reasons: (1) to keep legacy parameters understandable, and (2) rows is a more apt name for Som's adaptive sampling, where the number of samples is equal to the number of rows times the number of time steps.

Reply:
Thanks for the clarification!